README.md

```diff
@@ -1,2 +1,20 @@
-# team
-Docs &amp; Assets Related To The MozMeao Team
+# MozMeao
+
+## Core Values
+
+* We treat each other, co-workers, and our community with respect, trust, and honesty.
+* We strive to make every project a public resource, open and accessible to all.
+* We bring passion, fun, and creativity to our work. 
+* We ship often, we ship high quality.
+
+
+## Our Purpose 
+
+Our team provides technical expertise and development resources to Marketing.
```

Comment:
"Mozilla Marketing". I also think we can and should support other teams upon request. I know the Websites Durable Team strives for that.